mods/ra/rules/ai.yaml

```diff
@@ -82,7 +82,7 @@ Player:
 				CheckRadius: 10c0
 			Consideration@2:
 				Against: Enemy
-				Types: Vehicle, Tank, Infantry, Defense
+				Types: Vehicle, Tank, Infantry, Defense, Water
```

Comment:
I'm not convinced, so i'll need to see some evidence that this helps before giving a :+1:.